src/5e-SRD-Ability-Scores.json

```diff
@@ -1,145 +1,145 @@
-    ],
-    "skills": [
-      {
-        "url": "/api/skills/arcana",
-        "name": "Arcana"
-      },
-      {
-        "url": "/api/skills/history",
-        "name": "History"
-      },
-      {
-        "url": "/api/skills/investigation",
-        "name": "Investigation"
-      },
-      {
-        "url": "/api/skills/nature",
-        "name": "Nature"
-      },
-      {
-        "url": "/api/skills/religion",
-        "name": "Religion"
-      }
-    ],
-    "url": "/api/ability-scores/int"
-  },
-  {
-    "index": "wis",
-    "name": "WIS",
-    "full_name": "Wisdom",
-    "desc": [
-      "Wisdom reflects how attuned you are to the world around you and represents perceptiveness and intuition.",
-      "A Wisdom check might reflect an effort to read body language, understand someone's feelings, notice things about the environment, or care for an injured person. The Animal Handling, Insight, Medicine, Perception, and Survival skills reflect aptitude in certain kinds of Wisdom checks."
-    ],
-    "skills": [
-      {
-        "url": "/api/skills/animal-handling",
-        "name": "Animal Handling"
-      },
-      {
-        "url": "/api/skills/insight",
-        "name": "Insight"
-      },
-      {
-        "url": "/api/skills/medicine",
-        "name": "Medicine"
-      },
-      {
-        "url": "/api/skills/perception",
-        "name": "Perception"
-      },
-      {
-        "url": "/api/skills/survival",
-        "name": "Survival"
-      }
-    ],
-    "url": "/api/ability-scores/wis"
-  },
-  {
-    "index": "cha",
-    "name": "CHA",
-    "full_name": "Charisma",
-    "desc": [
-      "Charisma measures your ability to interact effectively with others. It includes such factors as confidence and eloquence, and it can represent a charming or commanding personality.",
-      "A Charisma check might arise when you try to influence or entertain others, when you try to make an impression or tell a convincing lie, or when you are navigating a tricky social situation. The Deception, Intimidation, Performance, and Persuasion skills reflect aptitude in certain kinds of Charisma checks."
-    ],
-    "skills": [
-      {
-        "url": "/api/skills/deception",
-        "name": "Deception"
-      },
-      {
-        "url": "/api/skills/intimidation",
-        "name": "Intimidation"
-      },
-      {
-        "url": "/api/skills/performance",
-        "name": "Performance"
-      },
-      {
-        "url": "/api/skills/persuasion",
-        "name": "Persuasion"
-      }
-    ],
-    "url": "/api/ability-scores/cha"
-  }
-]
+	{
+		"index": "str",
+		"name": "LANG-KEY-str",
+		"full_name": "LANG-KEY-strength",
+		"desc": [
+			"LANG-KEY-strength-measures-bodily-power-athletic-training-and-the-extent-to-which-you-can-exert-raw-physical-force",
```

Comment:
Should these things instead be something like `LANG-KEY-strength-description` and store that as a list instead? A key like what we have right now seems a bit silly.

Reply:
It would probably make your hash in `en.json` be a little less ridiculious? Maybe?

Reply:
> 
> 
> Should these things instead be something like `LANG-KEY-strength-description` and store that as a list instead? A key like what we have right now seems a bit silly.

Yes, this is totally what should happen. Having such big keys can be problematic on several fronts. The first and obvious one is that if typos are put in there, or on the contrary, fixed, we have to stick with what existed.

Reply:
If we want to keep the values the same type for everything, then the keys can instead be `LANG-KEY-strength-description-#`, where `#` is the index.